disperser/common/v2/blobstore/s3_blob_store.go

```diff
@@ -2,9 +2,9 @@ package blobstore
 
 import (
 	"context"
-	v2 "github.com/Layr-Labs/eigenda/core/v2"
 
 	"github.com/Layr-Labs/eigenda/common/aws/s3"
+	v2 "github.com/Layr-Labs/eigenda/core/v2"
```

Comment:
Is the `v2` alias necessary? Isn't the package name already `v2`?

Reply:
Going to alias it to corev2 for clarity.

Reply:
For some reason the linter didn't like changing it for one specific file `types_test.go` https://github.com/Layr-Labs/eigenda/actions/runs/11711503781/job/32620653412